changelog/fragments/support_configmap_compression.yaml

```
@@ -0,0 +1,19 @@
+# entries is a list of entries to include in
+# release notes and/or the migration guide
+entries:
+  - description: >
+      Allow compress the bundle content. Added a new cli flag `--gzip-configmap=true` to the `operator-sdk run bundle`.
```

Flag name changed in the implementation so we should make sure we change it in the changelog as well:
```suggestion
      Allow compress the bundle content. Added a new boolean cli flag `--gzip` to the `operator-sdk run bundle`.
```